_posts/2022-11-30-learning-day.md

```diff
@@ -0,0 +1,81 @@
+---
+title: The Learning Day Initiaitve
+date: 2022-11-30
+tags: [culture, learning-day, sticky]
+categories: [Machine Learning]
+image: assets/images/learning-day-blog/learning-day-dalle.png
+layout: post
+authors: [vinay-varma]
+latex: false
+---
+
+Greeting from the ML team at Skit!!!  Today we are announcing **The Learning Day Initiative**. 
```

Comment:
Should we 'announce'? Since we have been doing this for a while. We can maybe talk about this differently, maybe in a way where we are sharing our learnings with LDs and then next steps.